tests/test_transformers.py

```diff
@@ -799,82 +779,75 @@ def test_rare_feature_encoder_returns_correctly_dataframe(self):
             }
         )
         result = rare.transform(new_data)
+        assert new_data.shape == result.shape
 
-        test_data = pd.DataFrame(
-            {
-                "categorical": [99, 99, 99, "a", 99, 99, 3],
-                "numerical": [1, 2, 3, 4, 5, 6, 7],
-            }
-        )
-
-        assert isinstance(result, pd.DataFrame)
-        assert len(new_data) == len(result)
-        assert test_data.equals(result)
-
-    def test_rare_feature_encoder_handle_nan(self):
-        threshold = 2
-        fill_rare = "Rare"
-        rare = RareFeatureEncoder(threshold=threshold, fill_rare=fill_rare)
+    def test_rare_feature_encoder_returns_correctly_dataframe(
```

Comment:
Here returns correctly is more apt, since it's checking against a known, correct DataFrame :+1: